low-level/framework/sspl_rabbitmq_reinit

```diff
@@ -257,8 +257,8 @@ def _create_vhost_if_necessary(virtual_host, product):
             response = _send_command(command)
 
         vhosts = response.split('\n')
-        assert vhosts[0] == 'Listing vhosts ...'
         for vhost in vhosts[1:-1]:
+            print(f'vhost: {vhost}')
```

Comment:
printing vhosts before loop to all values in one go. loop is breaking on one condition, in which case wont see all configured values

Reply:
Done